lib/parameters.py

```diff
@@ -213,6 +213,31 @@ def gen_avc_forced_idr_parameters(spec, profiles):
   params = gen_avc_forced_idr_variants(spec, profiles)
   return keys, params
 
+def gen_avc_tcbrc_variants(spec, profiles):
+  for case, params in spec.items():
+    for variant in copy.deepcopy(params.get("variants", dict()).get("tcbrc", [])):
+      uprofile = variant.get("profile", None)
+      cprofiles = [uprofile] if uprofile else profiles
+      rcmode  = variant["rcmode"]
+      bitrate = variant.get("bitrate", None)
+      if "cqp" == rcmode:
+        assert bitrate == None, "We shouldn't set a value to bitrate for CQP."
+        variant.update(maxrate = None)
+        if "cbr" == rcmode:
+          variant.update(maxrate = bitrate)
+        elif "vbr" == rcmode:
+          variant.update(maxrate = bitrate * 2)
```

Comment:
Now I'm confused... previous discussions state that tcbrc is only valid for VBR low power mode with ldb.